backend/src/@types/models/Question.d.ts

```diff
@@ -6,12 +6,16 @@ interface QuestionAttributes {
 	courseId: CourseId;
 	title: string;
 	question: string;
+	participationPoints: number;
+	correctnessPoints: number;
 }
 
 interface QuestionCreationAttributes {
 	folderId: FolderId | null;
 	courseId: CourseId;
 	question: string;
 	title: string;
-	QuestionOptions: Array<QuestionOption>;
+	participationPoints?: number;
```

Comment:
I think these can all be required, the frontend should always be providing these things.